nautobot/docs/configuration/optional-settings.md

```diff
@@ -956,3 +922,50 @@ Environment Variables:
 * `NAUTOBOT_SHORT_TIME_FORMAT`
 * `NAUTOBOT_DATETIME_FORMAT`
 * `NAUTOBOT_SHORT_DATETIME_FORMAT`
+
+---
+
+## Environment-Variable-Only Settings
+
+!!! warning
+    The following settings are **only** configurable as environment variables, and not via `nautobot_config.py` or similar.
+
+---
+
+### GIT_SSL_NO_VERIFY
+
+Default: Unset
+
+If you are using a self-signed git repository, you will need to set the environment variable `GIT_SSL_NO_VERIFY="1"`
+in order for the repository to sync.
+
+!!! warning
+    This _must_ be specified as an environment variable. Setting it in `nautobot_config.py` will not have the desired effect.
+
+---
+
+### NAUTOBOT_ROOT
+
+Default: `~/.nautobot/`
+
+The filesystem path to use to store Nautobot files (Jobs, uploaded images, Git repositories, etc.).
+
+This setting is used internally in the core settings to provide default locations for [features that require file storage](../../configuration/#file-storage), and the [default location of the `nautobot_config.py`](../../configuration/#specifying-your-configuration).
+
+!!! warning
+    Do not override `NAUTOBOT_ROOT` in your `nautobot_config.py`. It will not work as expected. If you need to customize this setting, please always set the `NAUTOBOT_ROOT` environment variable.
+
+## Django Settings
+
+The below setting is documented for WSGI for clarity purposes.
+
+### FORCE_SCRIPT_NAME
+
+Default: `None`
+
+If not `None`, this will be used as the value of the `SCRIPT_NAME` environment variable in any HTTP request. This setting can be used to override the server-provided value of `SCRIPT_NAME`, which is most commonly used for hosting Nautobot in a subdirectory (e.g. _example.com/nautobot/_).
+
+!!! important
+    To host Nautobot under a subdirectory you must set this value to match the same prefix configured on your HTTP server. For example, if you configure NGINX to serve Nautobot at `/nautobot/`, you must set `FORCE_SCRIPT_NAME = "/nautobot/"`.
+
+Please see the [official Django documentation on `FORCE_SCRIPT_NAME`](https://docs.djangoproject.com/en/stable/ref/settings/#force-script-name) for more information.
```

Comment:
So why did this need to be moved to a separate section of its own, if it's a Django setting that you specify in `nautobot_config.py`?

Reply:
I am not tracking your question? My understanding is there are Nautobot Config settings that are here in one group and Django settings in another group, I would have that we should separate. 

Anything else you are hinting at, is not resonating with me now.

Reply:
My question is what's special/different about this setting that it needs to be separate from all the other configs? I'm not seeing why `FORCE_SCRIPT_NAME` (a Django setting that is configured in `nautobot_config.py`) is different than `ADMINS`, `DEBUG`, `HTTP_PROXIES`, `LOGGING`, etc. (also Django settings that are configured in `nautobot_config.py`). Why not leave it where it was?

Reply:
I think you are highlighting the confusion I am having with the documentation. It seems that we have a mix of Django settings (ADMINS, DEBUG, HTTP_PROXIES, LOGGING, etc), with Nautobot only settings (BRANDING_FILEPATHS, NAPALM_USERNAME, etc). Why are we documenting both? Why not document `CSRF_USE_SESSIONS` (which is a valid django setting) as an example.

There may be very good reasons to document django settings here, rather than refer to Django settings, but I do not understand what the decision tree is right now. I think either of the following are good steps forward:
* Split all of the Django settings from Nautobot settings
* Document why some Django settings are shown and others are not (e.g. the ones shown are most likely to cause issues)

Reply:
Fair points, but perhaps out of scope for the original stated purpose of this PR? :-) Can we scope this down to just addressing the environment-variables-as-distinct-from-config-variables issue, and get a follow up documentation/housekeeping issue open to revisit whether we should "undocument" some of the Django built-in configs variables?

Reply:
Updated created #2153 to track.